packages/wdio-sauce-service/src/launcher.js

```diff
@@ -51,32 +51,36 @@ export default class SauceLauncher {
             }
         }
 
-        if (Array.isArray(capabilities)) {
-            for (const capability of capabilities) {
-                if (!capability['sauce:options']) {
-                    capability['sauce:options'] = {}
-                }
+        function prepareCapability(capability) {
+            const isW3C = !(capability.platform || capability.version) || capability['sauce:options']
```

Comment:
There is `isW3C` method already defined in `packages/wdio-utils/src/envDetector.js` - we should use that and export it.

Reply:
I get why we would want to use `isW3C` (good idea!). Do you mind if I ask why we would want to also export that function from this package? I'm new to JavaScript so curious.

Reply:
I was suggesting to export it from the `@wdio/utils` package and import it here.